Core/StateManager.cs

```diff
@@ -1,25 +1,100 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Linq;
+using Core.TimeSeries;
 using Newtonsoft.Json;
 
 namespace Core
 {
 	public class StateManager
 	{
-		public static SortedSet<Transaction> Load()
+		public static void LoadFromJson(string stateJson)
 		{
-			if (!File.Exists("state.json"))
+			if (string.IsNullOrWhiteSpace(stateJson))
 			{
-				return new SortedSet<Transaction>();
+				return;
 			}
 
-			return JsonConvert.DeserializeObject<SortedSet<Transaction>>(File.ReadAllText("state.json"));
+			State.Instance = JsonConvert.DeserializeObject<State>(stateJson);
 		}
 
-		public static void Save()
+		public static string SaveToJson()
 		{
-			var transactions = JsonConvert.SerializeObject(State.Transactions);
-			File.WriteAllText("state.json", transactions);
+			return JsonConvert.SerializeObject(State.Instance);
 		}
+
+		public static void LoadTransactions(IEnumerable<Stream> files)
+		{
+			var importers = new List<TransactionsImporter>();
+			var inMemoryFiles = files.Select(f =>
+			{
+				var ms = new MemoryStream();
+				f.CopyTo(ms);
+				ms.Position = 0;
+				return ms;
+			}).ToList();
+			// todo: add all importers
+			// importers.Add(Privar);
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=5836065)